test/mocks/providers/provider_login_page.dart

```diff
@@ -0,0 +1,26 @@
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/database/user_repository_service.dart";
+import "package:proxima/views/navigation/routes.dart";
+import "package:proxima/views/pages/login/login_page.dart";
+
+import "../overrides/override_auth_providers.dart";
+import "../overrides/override_home_view_model.dart";
+
+ProviderScope loginPageProvider(FakeFirebaseFirestore fakeFireStore) {
+  final userRepo = UserRepositoryService(firestore: fakeFireStore);
+
+  return ProviderScope(
+    overrides: [
+      ...firebaseAuthMocksOverrides,
+      ...mockEmptyHomeViewModelOverride,
+      userRepositoryProvider.overrideWithValue(userRepo),
```

Comment:
By humble curiosity, how did you choose which overrides to extract in their own package, and which to keep in the providers? I have seen multiple overrides in various the providers.

Reply:
Good question!
I actually had a conversation with Yoann on our group about it.
It is because we reuse the parameters.

Attaching the screenshot :)
![image](https://github.com/ProximaEPFL/proxima/assets/101193378/d1753eb3-aceb-4579-977e-fd0ace9d30ec)